data/types/pumpstatus/device.go

```diff
@@ -0,0 +1,44 @@
+package pumpstatus
+
+import (
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	MinDeviceIDLength     = 1
+	MinModelLength        = 1
+	MinManufacturerLength = 1
```

Comment:
Should there also be Max for these? I've seen `1000` in other parts of `platform` for max string length.

Reply:
ok